src/views/createCommunity/index.tsx

```diff
@@ -507,7 +511,7 @@ function CreateCommunityScreen() {
                             config.locationErrorMargin,
                     },
                     email,
-                    coverMediaId: apiResult.data.data.id,
+                    coverMediaId: apiRequestResult.data.id,
```

Comment:
Is it not `apiRequestResult.data.data.id` ?

Reply:
Yes. My mistake. Fixed.